makuai-corp-site/content/contact.html

```diff
@@ -0,0 +1,35 @@
+---
+title: "Contact"
+date: 2022-09-04T15:35:00+09:00
+draft: false
+---
+<head>
+    <link rel="stylesheet" href="/css/contact.css">
+</head>
+<div class="contact">
+    <h1 class="contact__heading">問い合わせ</h1>
+    <p class="contact__text">プログラム支援、またはプログラミングスクールについて、以下のフォームよりお問い合わせください。</p>
+    <form class="contact__form">
+        <label for="name" class="contact__form-label">名前</label>
+        <div class="contact__form-input-outline">
+            <input type="text" id="name" placeholder="名前を入力してください" required class="contact__form-input contact__form-input--name">
+        </div>
+        <label for="email" class="contact__form-label">メール</label>
+        <div class="contact__form-input-outline">
+            <input type="email" id="email" placeholder="メールを入力してください" required class="contact__form-input contact__form-input--email">
+        </div>
+        <label for="message" class="contact__form-label">メッセージ</label>
+        <div class="contact__form-input-outline">
+            <textarea type="text" id="message" placeholder="メッセージを入力してください" required class="contact__form-input contact__form-input--message"></textarea>
+        </div>
+        <div class="contact__form-policy">
+            <p class="contact__form-policy-text">下記の個人情報の利用についての文書を必ずご確認ください。</p>
+            <a href="#" class="contact__form-policy-link"><strong>個人情報の取り扱いについて<span class="contact__form-policy-link--black">・</span>個人情報保護方針</strong></a>
```

Comment:
```suggestion
            <a href="/privacy" class="contact__form-policy-link"><strong>個人情報の取り扱いについて<span class="contact__form-policy-link--black">・</span>個人情報保護方針</strong></a>
```
ここはリンクを個人情報保護方針のページにしてもいいと思います。